src/main/docker/jre/Dockerfile

```diff
@@ -35,6 +35,12 @@ RUN microdnf -y install java-${JAVA_VERSION}-openjdk && \
 # Set JRE Home
 ENV JRE_HOME=/etc/alternatives/jre_${JAVA_VERSION}_openjdk
 
+# Update java.security file to fix SecureRandom.getInstanceStrong() not being available on a FIPS enabled system:
+# See: https://issues.redhat.com/browse/RHEL-3478
+RUN java_security_file=$(find /etc/java -name "java.security" | grep -v "jre" | sort -r | head -n 1) && \
```

Comment:
- Use `/etc/alternatives/jre_${JAVA_VERSION}_openjdk/conf/security/java.security`.  It will get you to the same location without searching for it.
- Let's make this a part of the earlier `RUN` statement so that it doesn't cause an extra layer in the image.